research/object_detection/eval_util.py

```diff
@@ -206,6 +206,8 @@ def _run_checkpoint_once(tensor_dict,
                          master='',
                          save_graph=False,
                          save_graph_dir='',
+                         inter_op=0,
```

Comment:
Please update doc string with these variables.

Reply:
thanks for the comments, i have added the inter_op/intra_op in the doc string already.